src/hooks/useScanApi.ts

```diff
@@ -343,8 +343,19 @@ const useDownloadReportApi = (onAddAlert: (alert: AlertProps) => void) => {
     [onAddAlert, t]
   );
 
+  const callbackErrorNoReport = useCallback(() => {
+    onAddAlert({
+      title: t('toast-notifications.description_report_downloaded_error_no_report'),
+      variant: 'danger'
+    });
+    return;
+  }, [onAddAlert, t]);
+
   const downloadReport = useCallback(
-    async (reportId: number) => {
+    async (reportId: number | undefined) => {
+      if (!reportId) {
+        return callbackErrorNoReport();
+      }
```

Comment:
Good idea! thanks for the feedback!

Reply:
replacing `isDisabled` with `disabled` is proving to be tricky. If I replace `isDisabled` with `disabled` it doesn't work 🤔 

Any idea why?

Reply:
https://github.com/quipucords/quipucords-ui/pull/513/files#r1834930552

Reply:
apologies for the confusion @bruno-fs ... `isDisabled` was a reference to what `disabled` was doing behind the scenes... I'll take another look since you appear to have updated/added something, but changing your original "disabled" to `disabled: scan.most_recent?.report_id !== undefined && scan.most_recent?.status !== 'completed',` would have produced the results you wanted

Reply:
Alright, no problem! 

I just added an extra commit that should hopefully pass all local testing - it will fail commit linting because it is a fixup! commit that I'll squash later if everything is OK.